docs/PUBLISHING.md

```diff
@@ -63,6 +63,8 @@ git push --follow-tags origin <branch> # this will also push tags
 
 For the commit message, use the new version's content in the [CHANGELOG.md](../CHANGELOG.md) (e.g., all the changes for version 1.2.3). Review the content in the changelog to make sure the notes, and the from-version and to-version are correct. Note the link to the storybook on this PR CI/CD. You will add this link to the GitHub release notes later.
 
+**NOTE**: if this is a hotfix release for an older version of EDS (not latest), skip to step 9.
```

Comment:
What about the scenario where the hotfix is for an older release of EDS but is still relevant to the latest version of EDS? Maybe as non urgent?

Reply:
I was tempted to dig into that, but it is a corner case. For now, I was thinking we could consider cherry-picking the change back into `next` or just re-implementing it? Pragmatically thinking to cross that bridge when we come to it :)

Reply:
That sounds good to me